websrc/source/documents/networking/services.md

```diff
@@ -17,16 +17,95 @@ Services offer:
 - Minimal downtime.
 
 A provider is one or more containers that match the label selector associated with the services.
+ **Note**: This feature would be deprecated since docker 1.12 onword service load balancer feature  is supported natively by docker swarm. 
 
 ## Defining Services
 
 After you create your network, you can define services using the UI, the `netctl` command line interface (CLI), or the Contiv Network REST APIs, but it is recommended
 to use the UI or API to take advantage of the authentication and authorization options in those interfaces.
 
-Note: Service requirements are defined by *selectors*. Selectors are key-value pairs 
+**Note**: Service requirements are defined by *selectors*. Selectors are key-value pairs 
 that group providers with matching labels. 
 
-To create a Service Load Balancer using the UI:
+###To create a Service Load Balancer using netctl:
```

Comment:
A space is required after `###` so it shows up as a heading properly

Reply:
Ok